### PR TITLE
Change requirements structure

### LINKED ISSUE #93: Setup better requirements structure
I am thinking about following something along [this line ](https://stackoverflow.com/questions/17803829/how-to-customize-a-requirements-txt-for-multiple-environments)

### PR DESCRIPTION
**Pull request by interesaaat**
Closes #93.

### CODE REVIEW

File: requirements/test.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-r common.txt
```
Comment (ksaur) [inline requirements/test.txt:1]: Ok, so the plan is to have this separate testing file, and then leave regular `requirements.txt` in place as an alternative to `pip install` yes?

File: .github/workflows/pythonapp.yml [hunk 2]
```diff
@@ -28,21 +28,18 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install -r requirements/test.txt .
```
Comment (ksaur) [inline .github/workflows/pythonapp.yml:31]: does the "." add something here?

**Review by ksaur — APPROVED** [review]

File: requirements/test.txt (hunk 1 above)
Comment (ksaur) [reply requirements/test.txt:1]: Ah ok, i see your comments  in #93

File: .github/workflows/pythonapp.yml (hunk 2 above)
Comment (interesaaat) [reply .github/workflows/pythonapp.yml:31]: I was surprised too. basically without "." it was not installing hummingbird, but only the requirements.

**Pull request merged**